docs/Networking.asciidoc

```diff
@@ -14,15 +14,15 @@ user networking which requires no further configuration.
 For more advanced setups or tests that require multiple jobs to be
 in the same networking the <<TAP based network,TAP>> or <<VDE Based Network,VDE>> based modes can be used.
 
-== Qemu user networking
+== Qemu User Networking
```

Comment:
Because in accordance with the Chicago Manual of Style (8.157 Principles of headline-style capitalization):

Capitalize the first and last words in titles and subtitles, and capitalize all other major words (nouns, pronouns, verbs, adjectives, adverbs, and some conjunctions).

The other thing is that I did not capitalize legacy titles (the titles, that were here before my changes). I will add capitalization to all other titles in my PR.